packages/web-styles/src/components/topic-teaser.scss

```diff
@@ -13,72 +18,96 @@
     }
   }
 
+  .topic-teaser-container {
+    @include media-breakpoint-up(xxl) {
+      padding-left: map.get(grid.$grid-container-padding, xxl) * 0.5;
+      padding-right: map.get(grid.$grid-container-padding, xxl) * 0.5;
+    }
+  }
+
   &:not(.topic-teaser-reverse) {
-    > .container {
-      @include media-breakpoint-up(lg) {
-        padding-right: 0;
+    @each $key in topic-teaser.$grid-keys-upper {
+      @include media-breakpoint-only($key) {
+        .container {
+          padding-right: 0;
+        }
       }
     }
 
-    .topic-teaser-container > .row {
-      margin-right: 0;
-    }
+    .topic-teaser-container {
+      @include media-breakpoint-up(xxl) {
+        padding-right: 0;
+        // webkit subpixel-issue fix
+        margin-right: -1px;
+        clip-path: inset(0 1px (map.get(spacing.$size-curve-huge, 'xxl') * -1) 0);
+      }
 
-    .topic-teaser-image-container {
-      padding-right: 0;
+      > .row {
+        justify-content: flex-end;
+      }
     }
 
-    .topic-teaser-content {
-      @include media-breakpoint-up(xxl) {
-        padding-left: 4.5rem;
+    .topic-teaser-image-container {
+      @each $key in topic-teaser.$grid-keys-lower {
```

Comment:
Similarly, @include media-breakpoint-down($grid-keys-lower[last item]) instead of the loop.

Reply:
In this case you are wrong. In each `$grid-keys-lower` breakpoint a `magin-left` or `margin-right` is set on the `.topic-teaser-image-container`, depending on the `grid.$grid-container-padding` variable.
So it is absolutely necessary to iterate over each of these breakpoints.